src/omero_cli_render.py

```diff
@@ -488,23 +458,21 @@ def edit(self, args):
                                   img, args.object,
                                   args.skipthumbs)
                 break
-            # finally:
-            # img._closeRE()
 
         if namedict:
             self._update_channel_names(gateway, iids, namedict)
 
-        # gateway._assert_unregistered("edit")
+    def edit(self, args):
+        self.ctx.err("WARNING: 'edit' command is deprecated, please "
```

Comment:
What's the target tag for this plugin? `0.1.1` or `0.2.0`? If the latter, how about make the breaking change to keep this clean, and change the message to `die`?

Reply:
Not sure, breaking or not? But even then, wouldn't be much cleaner, still need to keep the 'edit' command and method (although it just `die`s), don't I?

Reply:
I was thinking it forces any current users to change to the new command whilst providing a helpful message, in case we want to reuse `edit` in future. I'm not overly bothered though.